lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1755,7 +1804,7 @@ def cmd_sessions(*args)
       end
     when 'list', 'list_inactive', nil
       print_line
-      print(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, search_term: search_term))
+      print_status(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```

Comment:
Sorry dude I know I asked you to change this but it needs to go back, I thought it was a mistake but the tables shouldn't be `print_status` since it appends a little extra to the output
```suggestion
      print(Serializer::ReadableText.dump_sessions(framework, show_active: show_active, show_inactive: show_inactive, show_extended: show_extended, verbose: verbose, sessions: matching_sessions))
```